weave/bin/modules/cli_db_tools.py

```
@@ -0,0 +1,122 @@
+    weave db tool status
+    """
+    from .cli_migrate import check_and_install_tools
+    
+    # Run check without offering to install
+    console.print("[blue]🔍 Checking migration tools availability...[/blue]")
+    
+    tools_status = {}
+    
+    # Check Python dependencies
+    try:
+        import requests
```

Ruff (0.11.9)

`requests` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)